src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 4 failures in `PowerShell-CI-macos`
Error occurred in test script '/Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:155 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:168 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:181 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:194 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:206 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:213 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:225 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:241 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:255 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:155 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:168 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:181 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:194 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:206 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:213 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:225 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:241 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:255 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:155 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:168 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:181 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:194 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:206 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:213 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:225 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:241 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:255 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1'
```powershell
At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:155 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:168 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:181 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:194 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:206 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:213 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:225 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:241 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1:255 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```